### PR TITLE
Release notes for 1.7.6

### PR DESCRIPTION
**Pull request by JimmyCYJ**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[x] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/releases/1.7.x/announcing-1.7.6/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- /news/announcing-1.7.6
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.7.5 and Istio 1.7.6
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** Fixed an issue causing telemetry HPA settings to be overridden by the inline replicas. ([Issue #28916](https://github.com/istio/istio/issues/28916))
```
Comment (ericvn) [inline content/en/news/releases/1.7.x/announcing-1.7.6/index.md:18]: Remove the redundant Fixed at the start of each line. `**Fixed** Fixed an` -> `** Fixed** an`

File: content/en/news/releases/1.7.x/announcing-1.7.6/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+- **Fixed** Fixed an issue preventing delegate `VirtualService` change to trigger any xDS push. ([Issue #29123](https://github.com/istio/istio/issues/29123))
```
Comment (ericvn) [inline content/en/news/releases/1.7.x/announcing-1.7.6/index.md:20]:
```suggestion
- **Fixed** an issue where a delegate `VirtualService` change would not trigger an xDS push. ([Issue #29123](https://github.com/istio/istio/issues/29123))
```

File: content/en/news/releases/1.7.x/announcing-1.7.6/index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+- **Fixed** Fixed an issue that causing a very high memory usage with a large number of `ServiceEntry`. ([Issue #25531](https://github.com/istio/istio/issues/25531))
```
Comment (ericvn) [inline content/en/news/releases/1.7.x/announcing-1.7.6/index.md:22]:
```suggestion
- **Fixed** an issue that caused a very high memory usage with a large number of `ServiceEntry`s. ([Issue #25531](https://github.com/istio/istio/issues/25531))
```

**Comment by JimmyCYJ** [comment]
Please take a look. Thanks!

**Comment by ericvn** [comment]
@JimmyCYJ Changed the cherrypick as istio.io is showing what's in the `release-1.8` branch.

**Review by ericvn — APPROVED** [review]

**Pull request merged**